mflux_ai/core.py

```diff
@@ -198,3 +201,63 @@ def get_dataset(object_name, bucket_name="datasets"):
     in_memory_file.seek(0)
 
     return joblib.load(in_memory_file)
+
+
+def get_best_run(model_group_name):
+    """
+    Fetch the run id for the best run in a model group
+
+    :param model_group_name:
+    """
+
+    project_token = os.environ["MLFLOW_TRACKING_TOKEN"]
```

Comment:
If this environment variable is not set, please throw an exception with a sensible error message, maybe something like this: `Could not find the MFlux.ai project token for authentication. Please call mflux_ai.init("your_project_token_goes_here") before calling mflux_ai.get_best_run()`